src/components/AudioTrack/AudioThumb.vue

```diff
@@ -0,0 +1,18 @@
+<template>
+  <!-- Should have a fixed width -->
+  <div>
+    <div class="h-0 w-full pt-full bg-yellow">&nbsp;</div>
+  </div>
```

Comment:
Does this need a wrapping `div` or could it just be:

```suggestion
  <div class="h-0 w-full pt-full bg-yellow">&nbsp;</div>
```

Reply:
The `div` is needed because the aspect ratio hack doesn't work on `flex` children. I'm desperately waiting for general support for [CSS `aspect-ratio` property](https://caniuse.com/mdn-css_properties_aspect-ratio).

Reply:
But the responsibility of wrapping should be on the layout and not the thumbnail, so it's fixed in da4ff50.